moped-editor/src/views/projects/newProjectView/NewProjectView.js

```diff
@@ -113,105 +57,75 @@ const NewProjectView = () => {
   /**
    * Persists a new project into the database
    */
-  const handleSubmit = () => {
-    const newSignalError = fromSignalAsset && !Boolean(signal);
-    setSignalError(newSignalError);
+  const handleSave = (formData) => {
+    const {
+      isSignalProject,
+      signal,
+      projectName,
+      projectSecondaryName,
+      description,
+    } = formData;
+
+    const payload = {
+      project_name: projectName,
+      project_name_secondary:
+        projectSecondaryName.length > 0 ? projectSecondaryName : null,
+      project_description: description,
+      // Use potential phase as default
+      moped_proj_phases: {
+        data: [
+          {
+            phase_id: 1,
+            is_current_phase: true,
+            phase_start: new Date(new Date().setHours(0, 0, 0, 0)),
+            is_phase_start_confirmed: true,
+          },
+        ],
+      },
+    };
 
-    if (projectDetails.project_name.trim().length === 0) {
```

Comment:
I can save blank spaces as the project name (and as the project description) then I get an error 
```
NewProjectView.js:71 Uncaught (in promise) TypeError: Cannot read properties of null (reading 'length')
    at handleSave (NewProjectView.js:71:1)
```

Reply:
oh this error is unrelated to the empty string! it occurred because I left the secondary name blank

Reply:
@frankhereford heres the same error you reported

Reply:
thank y'all - I just pushed more code to:
- trim values before validating and saving
- null out secondary name if it is an empty string before the mutation happens